### PR TITLE
add index and HATEOS links for the REST API

### PR DESCRIPTION
**Pull request by yanns**
@hjr3 please review

### CODE REVIEW

File: src/mgmt.rs [hunk 1]
```diff
@@ -1,55 +1,117 @@
...
-use rustful::{Server, Context, Response, TreeRouter};
-use rustc_serialize::json;
+use rustful::{Server, Context, Response, StatusCode, TreeRouter};
+use rustful::header::ContentType;
+use rustc_serialize::Encodable;
+use rustc_serialize::json::{Encoder, EncodeResult};
 
 use pool::Pool;
 
+// HATEOS links: https://en.wikipedia.org/wiki/HATEOAS
```
Comment (Hoverbear) [inline src/mgmt.rs]: Should be HATEOAS no?

Comment (yanns) [inline src/mgmt.rs]: yes good catch

**Comment by hjr3** [comment]
Great! I am a big fan of hypermedia. I would eventually like to https://github.com/hjr3/hal-rs for this.

**Comment by yanns** [comment]
cool, I did not know about this library!

**Pull request merged**